packages/react-cosmos-component-playground/src/components/FixtureList/index.jsx

```diff
@@ -97,8 +101,17 @@ export default class FixtureList extends Component {
         </div>
         <div className={styles.list}>
           {components.map((component, i) => {
+            const componentClassNames = classNames(styles.component, {
+              [styles.componentSelected]:
+                component === urlParams.component &&
+                !urlParams.fixture,
+            });
             return (
-              <div key={i} className={styles.component}>
+              <div
+                key={i}
+                className={componentClassNames}
+                onClick={() => this.onComponentClick(component)}
```

Comment:
Wouldn't it make more sense to put the onClick handler on the `componentName` div? This way you don't need `stopPropagation` anymore as the fixture buttons will no longer be contained in the component button.